client/src/views/page-components/ChatDetailPage/ChatDetailPageFooter/ChatDetailPageFooter.js

```diff
@@ -0,0 +1,54 @@
+import { SEND_ICON } from '../../../../constants/imagePath.js';
+import { createDOMwithSelector } from '../../../../utils/createDOMwithSelector.js';
+
+export default function ChatDetailPageFooter({ $target, onSendChatHandler }) {
+  this.state = {
+    text: '',
+  };
+  this.$footer = createDOMwithSelector('div', '.chatDetailPageFooter');
+
+  $target.appendChild(this.$footer);
+  this.$footer.innerHTML = `
+      <input class="chatDetailPageFooter__input" placeholder="메시지를 입력하세요."/>
+      <button class="chatDetailPageFooter__button">
+        <img class="chatDetailPageFooter__icon" src="${SEND_ICON}"/>
+      </button>
+    `;
+
+  this.$sendButton = document.querySelector('.chatDetailPageFooter__button');
+
+  this.setState = (nextState) => {
+    this.state = nextState;
+    this.render();
+  };
+  this.render = () => {
+    if (this.state.text.length > 0) {
+      this.$sendButton.disabled = false;
+    } else {
+      this.$sendButton.disabled = true;
+    }
```

Comment:
아래처럼 할 수도 있을 것 같네요 🤣

```suggestion
    const isTextLengthOverZero = this.state.text.length > 0
    this.$sendButton.disabled = !isTextLengthOverZero;
```

```suggestion
    this.$sendButton.disabled = !(this.state.text.length > 0)
```

Reply:
오 코드가 이렇게 깔끔해질 수도 있군요...
하나 배워갑니다! 감사합니다! :)